packages/cli/lib/zeroYaml/init.ts

```diff
@@ -0,0 +1,104 @@
+import { exec } from 'node:child_process';
+import fs from 'node:fs';
+import path, { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { promisify } from 'node:util';
+
+import chalk from 'chalk';
+import ora from 'ora';
+
+import { printDebug } from '../utils.js';
+import { NANGO_VERSION } from '../version.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const execAsync = promisify(exec);
+
+/**
+ * Init a new nango folder
+ */
+export async function initZero({ absolutePath, debug = false }: { absolutePath: string; debug?: boolean }): Promise<boolean> {
+    printDebug(`Creating the nango integrations directory in ${absolutePath}`, debug);
+
+    const stat = fs.statSync(absolutePath, { throwIfNoEntry: false });
+
+    // Create directory if it doesn't exist
+    if (!stat) {
+        printDebug(`Directory does not exist`, debug);
+
+        await fs.promises.mkdir(absolutePath);
+    } else if (!stat.isDirectory()) {
+        console.log(chalk.red(`The path provided is not a directory. Exiting.`));
+        return false;
+    }
+
+    // Copy example folder
+    {
+        const spinner = ora({ text: 'Copy example' }).start();
+        const exampleFolderPath = path.join(__dirname, '..', '..', 'example');
+        try {
+            printDebug(`Copy example folder`, debug);
+
+            await fs.promises.mkdir(absolutePath, { recursive: true });
+            await copyRecursive(exampleFolderPath, absolutePath);
```

Comment:
`fs.cp` doesn't work? https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options

Reply:
it's still experimental in node 18 unfortunately